cmake/config/Config.cmake.in

```diff
@@ -188,3 +188,6 @@ SET(DEAL_II_TARGET_RELEASE "@CONFIG_TARGET_RELEASE@")
 
 # Full list of targets with "debug" and "optimized" keywords for easy use with TARGET_LINK_LIBRARIES:
 SET(DEAL_II_TARGET "@CONFIG_TARGET@")
+
+# The feature configuration file:
+SET(DEAL_II_TARGET_CONFIG "${DEAL_II_PATH}/${DEAL_II_PROJECT_CONFIG_RELDIR}/${DEAL_II_PROJECT_CONFIG_NAME}FeatureConfig.cmake")
```

Comment:
@tamiko 
did you mean to use `DEAL_II_FEATURE_CONFIG` here? To be used as
`INCLUDE(${DEAL_II_FEATURE_CONFIG}) # loads deal.IIFeatureConfig.cmake`